src/__init__.py

```diff
@@ -0,0 +1,13 @@
+# Copyright 2019 Google LLC
```

Comment:
It looks like all these years are wrong ;)

Reply:
Thanks for catching!